src/_P076_HLW8012.ino

```diff
@@ -738,16 +751,15 @@ void IRAM_ATTR p076_hlw8012_cf_interrupt() {
   }
 }
 
-const __FlashStringHelper* p076_getQueryString(uint8_t query) {
-  switch (query)
-  {
-    case 0: return F("Voltage_V");
-    case 1: return F("Current_A");
-    case 2: return F("Active Power_W");
-    case 3: return F("Reactive Power_VAR");
-    case 4: return F("Apparent Power_VA");
-    case 5: return F("Power_Factor_cosphi");
-    case 6: return F("Energy_Ws");
+const __FlashStringHelper* p076_getQueryString(uint8_t value_nr, bool displayString) {
+  switch (value_nr) {
+    case 0: return displayString ? F("Voltage") : F("V");
+    case 1: return displayString ? F("Current") : F("A");
+    case 2: return displayString ? F("Active Power") : F("W");
+    case 3: return displayString ? F("Reactive Power") : F("VAR");
+    case 4: return displayString ? F("Apparent Power") : F("VA");
+    case 5: return displayString ? F("Power Factor") : F("cosphi");
+    case 6: return displayString ? F("Energy") : F("Ws");
```

Comment:
I still recommend to restore the 4 original values names `Voltage`, `Current`, `Power` and `PowerFactor`, for backward compatibility.